bookmark.js

```diff
@@ -11,35 +12,40 @@ javascript:(function() {
   a.href = URL.createObjectURL(new Blob([`<!DOCTYPE html>
 <html>
 <head>
-  <title>Chat GPT Conversation</title>
+  <title>Chat GPT: ${title}</title>
   <meta name="generator" content="chatGPT Saving Bookmark"/>
 <style>
+body {
+    background-color: rgb(32,33,35);
+    color: rgb(236,236,241);
+    font-size: 16px;
+    font-family: Söhne,ui-sans-serif,system-ui,-apple-system,sans-serif,Helvetica Neue,Arial,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji;
```

Comment:
Can you remove this long font family? Please use:

```css
font-family: sans-serif
```

Reply:
Fixed.